templates/glance/config/glance-api-config.json

```diff
@@ -7,6 +7,12 @@
         "owner": "glance",
         "perm": "0600"
       },
+      {
+        "source": "/var/lib/config-data/merged/policy.yaml",
+        "dest": "/etc/glance/glance.conf.d/policy.yaml",
```

Comment:
Thank @konan-abhi, now I can see the right policy file being used!
I'm just wondering if a change in the README can be added to this PR so we can make sure we document how to build projects and interact w/ glance